app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -353,7 +341,7 @@ function replaceSelectedText(textArea, text) {
 
     // IE 8-10
     else if(document.selection) {
-      const ieRange = document.selection.createRange();
+      t ieRange = document.selection.createRange();
```

Comment:
is this a typo?

Reply:
ummm... definitely. good catch. reverting this guy.